### ISSUE TITLE
Review feature categorizations

### ISSUE THREAD

**Issue by mattgarrish** [w3c/a11y-discov-vocab#10]
We should review the titles and categorization of the terms, as I believe this was added to the wiki only to help break up the long list of values.

I was skimming the values and noticed some, at least, may be misplaced -- highContrastAudio isn't necessarily in a transformed state or able to be transformed, for example, and neither arguably is large text print (it may be the only way it's published and large print isn't necessarily intended for further transformation).

**Comment by clapierre** [comment]
Large Print I agree should be handled by displayTransformability as this was something when you had a physical print book that was produced with large fonts for low vision users.  Not sure if it would be useful for a fixed layout low vision book with the default font size being huge or images containing large font text.

**Comment by mattgarrish** [comment]
> Large Print I agree should be handled by displayTransformability as this was something when you had a physical print book that was produced with large fonts for low vision users.

Being able to enlarge the font is display transformability, for sure, and I think that's the potential confusion of having them in close proximity under this heading.

The largePrint value is meant for books that come preformatted in large print. They could also be transformable into even larger text (or smaller) so you could have that value paired with displayTransformability, adding more confusion.
